roles/setup-services/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 - name: Wait for all services to be ready
   shell: oc get serviceinstance -o=jsonpath='{.items[*].status.conditions[0].status}' -n {{namespace}} | tr ' ' '\n' | grep True | wc -w
   register: ready_status_output
-  until: ready_status_output.stdout == "6"
+  until: ready_status_output.stdout.find('6') != -1
```

Comment:
The previous condition was never satisfied for me using macOS 10.14.5 and ansible-playbook 2.7.6. It would always hit 100 attempts - this resolved it.